build-tools/packages/build-tools/src/bumpVersion/README.md

```diff
@@ -1,4 +1,4 @@
-# Bump Version tool
+# Bump Version tool (deprecated)
```

Comment:
Not sure this was part of this PR? It's docs, I don't feel strongly about moving it, just pointing it out.